lib/ruby-lint/constant_path.rb

```diff
@@ -56,9 +56,10 @@ def resolve(scope)
     end
 
     ##
-    # Returns the very first segment of the constant path as an AST node.
+    # Returns the very first segment of the constant path as a pair type+name
+    # FIXME: unused now
     #
-    # @return [RubyLint::AST::Node]
+    # @return [Array(Symbol,String)]
```

Comment:
These changes don't seem to be related to this PR. If this method is not used anymore a separate PR should just remove it.